packages/mobile/src/page/Notice/Detail/Footer/index.tsx

```diff
@@ -18,8 +18,8 @@ function Footer({ articleId, isBookmark, isCouncil }: Props) {
   const addArticleBookmark = useAddArticleBookmark();
   const removeArticleBookmark = useRemoveArticleBookmark();
 
-  const handleCopy = async () => {
-    await navigator.clipboard.writeText(window.location.href);
+  const handleCopy = () => {
+    baseApp.postMessage(window.location.href);
```

Comment:
`baseApp` 객체가 있을때만 동작되도록 하는게 좋을것 같아요
```suggestion
    if(baseApp) baseApp.postMessage(window.location.href);
```